tck/src/test/java/org/eclipse/microprofile/jwt/tck/config/jwe/PrivateKeyAsJWKClasspathTest.java

```diff
@@ -103,9 +107,10 @@ public static WebArchive createLocationDeployment() throws IOException {
     public void testKeyAsLocation() throws Exception {
         Reporter.log("testKeyAsLocation, expect HTTP_OK");
 
+        PrivateKey signingKey = TokenUtils.readPrivateKey("/privateKey4k.pem");
         PublicKey publicKey = TokenUtils.readJwkPublicKey("/encryptorPublicKey.jwk");
         String kid = "mp-jwt";
-        String token = TokenUtils.encryptClaims(publicKey, kid, "/Token1.json");
+        String token = TokenUtils.signEncryptClaims(signingKey, null, publicKey, kid, "/Token1.json", false);
```

Comment:
@ayoho the last parameter has to be `true` - otherwise the test must fail because with the inner-signed tokens being encrypted, the JWE `cty` header must be set to `JWT` - this is clarified in the spec I believe and the existing sign/encrypt TCK tests enforce it, see also
https://github.com/eclipse/microprofile-jwt-auth/blob/master/tck/src/main/java/org/eclipse/microprofile/jwt/tck/util/TokenUtils.java#L452
thanks

Reply:
You're right, I had missed that part of the spec. I've pushed a new commit that changes these values to `true`.

Reply:
@ayoho thanks, it looks good. Can you please squash your 2 commits ?